make/config.mk

```diff
@@ -97,6 +93,32 @@ else
 USE_BLAS = atlas
 endif
 
+# whether use lapack during compilation
+# only effective when compiled with blas versions openblas/apple/atlas/mkl
+USE_LAPACK = 1
+
+# Name of separate lapack library
+LAPACK_LIB = liblapack.a
```

Comment:
About the LAPACK_LIB? I need the name of the library that must be present when looking into the subdirectories the lines below. Or do you mean something else?